appveyor.yml

```diff
@@ -16,10 +16,8 @@ init:
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
 
   - ECHO "Updating Environment"
-  - python -m pip install -U setuptools
   - python -m pip install -U pip
-  - python -m pip install -U wheel

```

Comment:
it seems both are needed for a after test action,, i dont recall why that one was necessary however

Reply:
Hmm. That's odd. I looked for steps in the appveyor.yml that relied on it, and I didn't see any... and the tests seem to fail within the test run, not after. I'll investigate and see if I can work out a solution for those failing tests so that the test suite isn't dependent on factors outside of tox.

Reply:
Nevermind, I [see it now](https://github.com/pypa/setuptools_scm/blob/8a69f1ce3ab532e1fd1bfb081c9932c1063c96d3/appveyor.yml#L39).